x-pack/test/case_api_integration/common/lib/utils.ts

```diff
@@ -556,6 +567,72 @@ export const ensureSavedObjectIsAuthorized = (
   entities.forEach((entity) => expect(owners.includes(entity.owner)).to.be(true));
 };
 
+export const createCaseWithConnector = async ({
```

Comment:
This is used by two different files now so I moved it here to make it more easily shared.